gempy/utils/stochastic_surface.py

```diff
@@ -0,0 +1,198 @@
+"""
```

Comment:
This file is breaking a test for this file. Since has absolutely nothing to do with viz I am going to remove the file and the test from the pull request. @pytzcarraldo just fix it and make a new one